thirdparty/erlang/zimbra-erlang/rpm/SPECS/erlang.spec

```diff
@@ -0,0 +1,206 @@
+Summary:	Zimbra's erlang
+Name:		zimbra-erlang
+Version:	VERSION
+Release:	1zimbra8.8b1ZAPPEND
```

Comment:
just curious, what does this 1zimbra8.8b1 signifies?

Reply:
it's just major and minor versions